keyboards/ymdk/ymd09/config.h

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2020 Patrick Fruh
+Copyright 2023 SHVD3x
```

Comment:
Original copyright shouldn't be removed. 

```suggestion
Copyright 2020 Patrick Fruh
Copyright 2023 SHVD3x
```